data/scopes/fanvil-X210.ini

```
@@ -0,0 +1,35 @@
+[metadata]
+scopeType = "model"
+displayName = "Fanvil X210"
+version = 4
+
+[data]
+cap_ldap_tls_blacklist =
+provisioning_url_filename = "$mac.cfg"
+tmpl_phone = "fanvil-X5.tmpl"
+fanvil_sidekey_count = 10
```

Add the new variable here!

```suggestion
fanvil_sidekey_count = 10
fanvil_lkpages_count = 3
```